compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/PlistSanityCheck.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.uikit
+
+import kotlinx.atomicfu.atomic
+import platform.Foundation.NSBundle
+import platform.Foundation.NSNumber
+import platform.darwin.DISPATCH_QUEUE_PRIORITY_LOW
+import platform.darwin.dispatch_async
+import platform.darwin.dispatch_get_global_queue
+
+internal object PlistSanityCheck {
+    private var isPerformed = atomic(false)
+
+    fun performIfNeeded() {
+        if (isPerformed.compareAndSet(expect = false, update = true)) {
+            dispatch_async(dispatch_get_global_queue(
+                DISPATCH_QUEUE_PRIORITY_LOW.toLong(),
+                0u
+            )) {
+                val entry = NSBundle
+                    .mainBundle
+                    .objectForInfoDictionaryKey("CADisableMinimumFrameDurationOnPhone") as? NSNumber
+
+                if (entry?.boolValue != true) {
+                    println("WARNING: `Info.plist` doesn't have a valid `CADisableMinimumFrameDurationOnPhone` entry. Framerate will be restricted to 60hz on iPhones. To support high frequency rendering on iPhones, add `CADisableMinimumFrameDurationOnPhone` entry to `Info.plist` and set it to YES.")
```

Comment:
```suggestion
                    println("WARNING: `Info.plist` doesn't have a valid `CADisableMinimumFrameDurationOnPhone` entry. Framerate will be restricted to 60hz on iPhones. To support high frequency rendering on iPhones, add `<key>CADisableMinimumFrameDurationOnPhone</key><true/>` entry to `Info.plist`.")
```

Can we write it this way, as in https://developer.apple.com/documentation/quartzcore/optimizing_promotion_refresh_rates_for_iphone_13_pro_and_ipad_pro#3885321?

Reply:
They can also set it using the visual editor (in case their Info.plist is generated from within xcodeproj). So I'm not sure if it's needed.
<img width="1070" alt="Screenshot 2023-09-07 at 12 21 21" src="https://github.com/JetBrains/compose-multiplatform-core/assets/4167681/7bf0c52c-44b0-4863-bbe5-8ce8d0c470c6">